client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -106,6 +107,11 @@ export const useStoreItemInfo = ( {
 		[ getIsIncludedInPlan, getIsOwned, sitePlan ]
 	);
 
+	const getIsExternal = useCallback(
+		( item: SelectorProduct ) => EXTERNAL_PRODUCTS_LIST.includes( item.productSlug ),
+		[]
+	);
```

Comment:
Since it doesn't have any dependencies, it doesn't need to be inside the hook, it can be outside the hook just like `getIsDeprecated`.

Reply:
this one has been updated. 👍 thanks